components/bl-gallery-component/src/components/gallery.js

```diff
@@ -0,0 +1,50 @@
+import LightBox from '../lib/lightbox-jquery.js';
+import { GalleryImage } from './gallery-image.js';
+
+export function Gallery({ images, options }) {
+  const {
+    alwaysShowNavOnTouchDevices,
+    imageLabel,
+    separator,
+    disableScrolling,
+    fadeDuration,
+    imageFadeDuration,
+    positionFromTop,
+    resizeDuration,
+    showImageNumberLabel,
+    wrapAround,
+  } = options;
+
+  LightBox.option({
+    'alwaysShowNavOnTouchDevices': alwaysShowNavOnTouchDevices,
+    'albumLabel': `${imageLabel} %1 ${separator} %2`,
+    'disableScrolling': disableScrolling,
+    'fadeDuration': fadeDuration,
+    'imageFadeDuration': imageFadeDuration,
+    'positionFromTop': positionFromTop,
+    'resizeDuration': resizeDuration,
+    'showImageNumberLabel': showImageNumberLabel,
+    'wrapAround': wrapAround,
+  });
+
+  return (
+    <div className="bl-customComponent-lightBox">
+      {
+        images.map(image => {
+          const { title, url, height } = image;
+
+          return (
+            <a
+              key={ url }
+              data-lightbox="example"
```

Comment:
What does mean example here?

Reply:
To group images in gallery we need to use the same attribute value for every image

Reply:
So we could change it to some special? Like 'bl-gallery' or something like that?